src/libmld/ldfsymorder.c

```diff
@@ -1,281 +1,96 @@
-__asm__(R""(
-.macro glabel label
-    .global \label
-    .balign 4
-    \label:
-.endm
+#include "libmld.h"
+
+/*
+# 0048D0E0 st_readst
```

Comment:
please remove the #'s
```suggestion
0048D0E0 st_readst
```